director/template/var_kv_test.go

```diff
@@ -56,5 +56,13 @@ var _ = Describe("VarKV", func() {
 				Expect(val3).To(Equal(map[interface{}]interface{}{"key31": "str"}))
 			})
 		})
+
+		Context("When key/value has newlines", func() {
```

Comment:
Might as well specify in the context that the string is not a yaml

Reply:
👍